requirements/testing.txt

```diff
@@ -2,6 +2,6 @@
 pytest-flask==1.0.0
 pytest-cov==2.7.1
 pytest-xdist==1.28.0
-requests==2.21.0
+requests>=2.21.0
```

Comment:
Probably better to do:

`~=` on all occurences of `>=` that you introduced.

You can check which exact dep it installed by running a:

`pip freeze|grep -i package`